remix-lib/src/execution/txHelper.js

```diff
@@ -41,13 +41,17 @@ module.exports = {
   },
 
   sortAbiFunction: function (contractabi) {
+    // Check if function is constant (introduced with Solidity 0.6.0)
+    function isConstant (funcABI) {
```

Comment:
Nothing wrong here. But if you want you can also do that : 
```typescript
const isConstant = ({ stateMutability }) => stateMutability === 'view' || stateMutability === 'pure'
```

Reply:
done